test/e2e/tilesets-import.test.js

```diff
@@ -40,8 +38,20 @@ function isValidVectorLayersValue(vectorLayers) {
   return vectorLayers.every((layer) => layer.id && layer.fields)
 }
 
+/**
+ * @param {ReturnType<typeof createServer>} server
+ * @param {string} importId
+ * @returns {Promise<void>}
+ */
+async function importCompleted(server, importId) {
+  await discard(server.getImportProgress(importId))
```

Comment:
similar comment about avoiding the helper. maybe not as curious about this case though

Reply:
Addressed in c3631bcd2d73aaaf727f2e5d113ffd4379cce222.